package.json

```diff
@@ -28,8 +28,8 @@
     "commander": "~2.15.0",
     "js-yaml": "~3.11.0",
     "length-prefixed-stream": "~1.5.1",
-    "limitd-protocol": "limitd/protocol#v2.0.5",
-    "limitdb": "^2.6.1",
+    "limitd-protocol": "limitd/protocol#v2.1.0",
+    "limitdb": "limitd/limitdb#54374ee",
```

Comment:
Remember to change this one to the correct dependency

Reply:
@elbuo8 Can you please cut a new release of limitdb? I had a pull request that got merged over there, but I'd like to point this dependency to NPM instead of Github.